configuration-client/package.json

```diff
@@ -52,16 +52,16 @@
       "last 1 safari version"
     ]
   },
-  "packageManager": "yarn@3.2.1",
+  "packageManager": "yarn@3.2.2",
   "devDependencies": {
     "@testing-library/jest-dom": "^5.16.4",
     "@testing-library/react": "^13.3.0",
-    "@testing-library/user-event": "^13.5.0",
-    "@types/jest": "^27.5.1",
+    "@testing-library/user-event": "^14.4.2",
+    "@types/jest": "^28.1.6",
     "@types/lodash": "^4.14.182",
-    "@types/node": "^16.11.36",
-    "@types/react": "^18.0.14",
-    "@types/react-dom": "^18.0.5",
+    "@types/node": "^18.6.5",
```

Comment:
The major version of this package should be in sync with the Node version or we will get potentially wrong types (not that it matters here for our frontend code, but it's a good practice)